src/main/java/seedu/address/model/util/SampleDataUtil.java

```diff
@@ -35,7 +39,7 @@ public static Person[] getSamplePersons() {
                        new Gender("M"),
                        getTagSet("friends"),
                        new Priority(),
-                       getAppointmentSet("Dental:2024-12-25:1235-1300")
+                       getAppointmentSet("Dental:2024-12-25:1235-1300"), EMPTY_MEDCON
```

Comment:
Sample data should not use EMPTY_MEDCON. Require proper sample data.

Reply:
I will amend too this for my priority portion, priority is default NONE for sample data. Will settle it in next PR.